src/widgets/launcher.rs

```diff
@@ -239,6 +241,16 @@ impl Widget for Launcher {
                     .map(|(x, y)| (x.unwrap(), y.clone()))
                     .collect::<Vec<(i64, Desktop)>>();
 
+                for desktop in self.options.iter() {
```

Comment:
There's probably a better way to do this in a single loop together with the name matches or in a more idiomatic way but it wasn't immediately obvious to me.

Reply:
Thought: Change the iterator above to return `(score, gnome_score, desktop)` tuples, sort using the biggest one below, and then in draw, either show `name`, or `name (gnome_name)`, depending on whether the gnome name had a score.

Dunno if that'll look/feel better. Just a thought. Could also be used for keyword and/or alternate language matching.